### DISCUSSION TITLE
安卓版蓝灯无法登录设备

### DISCUSSION THREAD

**Discussion by Jameshening** [getlantern/lantern-forum-cn#731] [Q&A]
ticket：74855498
7月8日购买了专业版，付款凭证如下：
![_cgi-bin_mmwebwx-bin_webwxgetmsgimg__ MsgID=4945242323406551657 skey=@crypt_5b34820a_73f6e29d7ca889429ba3152fe814df8d mmweb_appid=wx_webfilehelper](https://github.com/user-attachments/assets/ea48667c-f35b-4259-a8f6-279477f3698e)

邮箱为38790292@qq.com. 目前发现“通过电子邮件授权设备”登录专业版无效，点“提交”后界面一直转圈，邮箱也没有受到PIN码。

是否可以发激活码过来？我记得付款后就没收到过激活码

**Comment by MaxMKMax** [comment]
@Jameshening 
嗯嗯. 好的, 给我们几天工作日的时间. 系统在维修要后天才能发激活码. 谢谢您的等待.

**Reply by Jameshening** [reply]
什么时候能发过来？都快一周了

**Reply by Jameshening** [reply]
> @Jameshening
> 嗯嗯. 好的, 给我们几天工作日的时间. 系统在维修要后天才能发激活码. 谢谢您的等待.

@Mike777Khoo 请问什么时候能发激活码？

**Reply by Jameshening** [reply]
> 什么时候能发过来？都快一周了



> > @Jameshening
> > 嗯嗯. 好的, 给我们几天工作日的时间. 系统在维修要后天才能发激活码. 谢谢您的等待.
> 
> @Mike777Khoo 请问什么时候能发激活码？

什么时候能发激活码？

**Reply by Jameshening** [reply]
> @Jameshening
> 嗯嗯. 好的, 给我们几天工作日的时间. 系统在维修要后天才能发激活码. 谢谢您的等待.

什么时候能发？系统还没修好吗？
